Source/NSFileManager.m

```diff
@@ -2032,6 +2026,7 @@ - (NSDictionary*) fileSystemAttributesAtPath: (NSString*)path
 
   if (!GetVolumePathNameW(lpath, volumePathName, 128))
     {
+      *error = [NSError _last];
```

Comment:
I think there are other places in the code that do not check for this, but I'll investigate further.

In any case, having `*error` set to `NULL` is not wrong (although it would be quite misleading). In this case, what should we do? Generate a new `NSError` right here? What should it contain?